Flaky TestOracleIntegrationSmokeTest

```
2020-07-09T03:02:32.8048379Z [ERROR] Tests run: 256, Failures: 2, Errors: 0, Skipped: 205, Time elapsed: 388.896 s <<< FAILURE! - in TestSuite
2020-07-09T03:02:32.8053632Z [ERROR] init(io.prestosql.plugin.oracle.TestOracleIntegrationSmokeTest)  Time elapsed: 34.051 s  <<< FAILURE!
2020-07-09T03:02:32.8055567Z java.lang.RuntimeException: 
2020-07-09T03:02:32.8061832Z java.sql.SQLSyntaxErrorException: ORA-01940: cannot drop a user that is currently connected
2020-07-09T03:02:32.8062106Z 
2020-07-09T03:02:32.8064274Z 	at io.prestosql.plugin.oracle.TestingOracleServer.<init>(TestingOracleServer.java:73)
2020-07-09T03:02:32.8071240Z 	at io.prestosql.plugin.oracle.BaseOracleIntegrationSmokeTest.createQueryRunner(BaseOracleIntegrationSmokeTest.java:41)
2020-07-09T03:02:32.8076017Z 	at io.prestosql.testing.AbstractTestQueryFramework.init(AbstractTestQueryFramework.java:76)
2020-07-09T03:02:32.8080756Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-07-09T03:02:32.8086598Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-07-09T03:02:32.8092439Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-07-09T03:02:32.8095732Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-07-09T03:02:32.8101747Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2020-07-09T03:02:32.8107409Z 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
2020-07-09T03:02:32.8113908Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
2020-07-09T03:02:32.8119769Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
2020-07-09T03:02:32.8129982Z 	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
2020-07-09T03:02:32.8135641Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
2020-07-09T03:02:32.8143227Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2020-07-09T03:02:32.8148391Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2020-07-09T03:02:32.8152350Z 	at java.base/java.lang.Thread.run(Thread.java:834)
2020-07-09T03:02:32.8161541Z Caused by: java.sql.SQLSyntaxErrorException: ORA-01940: cannot drop a user that is currently connected
2020-07-09T03:02:32.8161694Z 
2020-07-09T03:02:32.8167677Z 	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:509)
2020-07-09T03:02:32.8173479Z 	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:461)
2020-07-09T03:02:32.8177958Z 	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1104)
2020-07-09T03:02:32.8182298Z 	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:550)
2020-07-09T03:02:32.8186347Z 	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:268)
2020-07-09T03:02:32.8190401Z 	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:655)
2020-07-09T03:02:32.8194488Z 	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:229)
2020-07-09T03:02:32.8200994Z 	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:41)
2020-07-09T03:02:32.8202474Z 	at oracle.jdbc.driver.T4CStatement.executeForRows(T4CStatement.java:928)
2020-07-09T03:02:32.8208253Z 	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1205)
2020-07-09T03:02:32.8211394Z 	at oracle.jdbc.driver.OracleStatement.executeInternal(OracleStatement.java:1823)
2020-07-09T03:02:32.8215776Z 	at oracle.jdbc.driver.OracleStatement.execute(OracleStatement.java:1778)
2020-07-09T03:02:32.8221257Z 	at oracle.jdbc.driver.OracleStatementWrapper.execute(OracleStatementWrapper.java:303)
2020-07-09T03:02:32.8226653Z 	at io.prestosql.plugin.oracle.TestingOracleServer.<init>(TestingOracleServer.java:70)
2020-07-09T03:02:32.8227382Z 	... 15 more
2020-07-09T03:02:32.8238190Z Caused by: Error : 1940, Position : 0, Sql = DROP USER apex_040000 CASCADE, OriginalSql = DROP USER apex_040000 CASCADE, Error Msg = ORA-01940: cannot drop a user that is currently connected
2020-07-09T03:02:32.8238312Z 
2020-07-09T03:02:32.8242531Z 	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:513)
2020-07-09T03:02:32.8243232Z 	... 28 more
2020-07-09T03:02:32.8243420Z 
2020-07-09T03:02:32.8250033Z [ERROR] destroy(io.prestosql.plugin.oracle.TestOracleIntegrationSmokeTest)  Time elapsed: 0.001 s  <<< FAILURE!
2020-07-09T03:02:32.8251921Z java.lang.NullPointerException
2020-07-09T03:02:32.8258069Z 	at io.prestosql.plugin.oracle.BaseOracleIntegrationSmokeTest.destroy(BaseOracleIntegrationSmokeTest.java:51)
2020-07-09T03:02:32.8265567Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-07-09T03:02:32.8273550Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-07-09T03:02:32.8280713Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-07-09T03:02:32.8284317Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-07-09T03:02:32.8290552Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2020-07-09T03:02:32.8295060Z 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
2020-07-09T03:02:32.8299802Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
2020-07-09T03:02:32.8304289Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
2020-07-09T03:02:32.8311729Z 	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
2020-07-09T03:02:32.8316018Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
2020-07-09T03:02:32.8322036Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2020-07-09T03:02:32.8327647Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2020-07-09T03:02:32.8330848Z 	at java.base/java.lang.Thread.run(Thread.java:834)
```

--- a/presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestingOracleServer.java
+++ b/presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestingOracleServer.java
@@ -14,6 +14,7 @@
 package io.prestosql.plugin.oracle;
 
 import org.testcontainers.containers.OracleContainer;
+import org.testcontainers.utility.MountableFile;
 
 import java.io.Closeable;
 import java.io.IOException;
@@ -37,6 +38,8 @@ public class TestingOracleServer
     public TestingOracleServer()
     {
         super("wnameless/oracle-xe-11g-r2");
+
+        withCopyFileToContainer(MountableFile.forClasspathResource("init.sql"), "/docker-entrypoint-initdb.d/init.sql");
 
         start();
 
@@ -67,7 +70,6 @@ public class TestingOracleServer
             statement.execute(format("CREATE USER %s IDENTIFIED BY %s DEFAULT TABLESPACE %s", TEST_USER, TEST_PASS, TEST_TABLESPACE));
             statement.execute(format("GRANT UNLIMITED TABLESPACE TO %s", TEST_USER));
             statement.execute(format("GRANT ALL PRIVILEGES TO %s", TEST_USER));
-            statement.execute("DROP USER apex_040000 CASCADE"); // Drop apex_040000 because it has 601 tables and leads to slow tests
         }
         catch (SQLException e) {
             throw new RuntimeException(e);

--- a/presto-oracle/src/test/resources/init.sql
+++ b/presto-oracle/src/test/resources/init.sql
@@ -1,0 +1,2 @@
+/* Drop apex_040000 because it has 601 tables and leads to slow tests */
+DROP USER apex_040000 CASCADE;

